nautobot_floor_plan/templatetags/seed_helpers.py

```diff
@@ -36,3 +37,53 @@ def count_children_floor_plans(location):
     """Returns count of Children with FloorPlans for a given location."""
     count = location.children.filter(floor_plan__isnull=False).count()
     return count
+
+
+@register.filter()
+def get_fieldset_field(form, field_name):
+    """Retrieve a field from the form using a dynamic field name."""
+    try:
+        return form[field_name]
+    except KeyError:
+        return None
+
+
+@register.filter
+def render_origin_seed(obj, axis):
+    """Render custom seed info for the specified axis if it exists, otherwise display the default seed."""
+    custom_label = obj.custom_labels.filter(axis=axis.upper()).first()
+    if custom_label:
+        try:
+            converter = label_converters.LabelConverterFactory.get_converter(custom_label.label_type)
+            # Convert and display the custom label start
+            numeric_value = converter.to_numeric(custom_label.start_label)
+            display_label = converter.from_numeric(numeric_value)
```

Comment:
Why do we need to convert the `start_label` back and forth like this? Can we not just do `display_label = custom_label.start_label`? A comment might be helpful.